FormalConjectures/ErdosProblems/330.lean

```diff
@@ -0,0 +1,67 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import FormalConjectures.Util.ProblemImports
+
+# Erdős Problem 330
+
+*Reference:* [erdosproblems.com/330](https://www.erdosproblems.com/330)
+-/
+
+namespace Erdos330
+
+open Filter Set
+open scoped BigOperators
+
+/-- `Rep_h A m` means `m` is a sum of exactly `h` terms from `A`. -/
+def Rep_h (A : Set ℕ) (h m : ℕ) : Prop :=
+  ∃ (f : Fin h → ℕ), (∀ i, f i ∈ A) ∧ (∑ i, f i) = m
+
+/--
+Use the **additive** version of your generic API:
+
+If your to_additive generated the name `Set.IsAsymptoticAddBasisOfOrder`, this alias
+plugs it in for `A : Set ℕ`. If your project used a different additive name, just
+swap that identifier below (only here).
```

Comment:
Why using "your" here? If you aer using a coding agent to write the code here, please make sure that distracting traces of that are removed. We don't need part of the prompting conversation in the end result.